deps/llvm.mk

```diff
@@ -448,6 +448,15 @@ $$(LLVM_BUILDDIR_withtype)/build-compiled: $$(LLVM_SRC_DIR)/$1.patch-applied
 LLVM_PATCH_PREV := $$(LLVM_SRC_DIR)/$1.patch-applied
 endef
 
+define LLVM_PROJ_PATCH
```

Comment:
ah nice! I have been using `git format-patch -1 --no-prefix` but this is cleaner